examples/nlp/language_modeling/megatron_ckpt_to_nemo.py

```diff
@@ -143,7 +143,15 @@ def convert(local_rank, rank, world_size, args):
     )
 
     if args.model_type == 'gpt':
+        # model = MegatronGPTModel.load_from_checkpoint(checkpoint_path, hparams_file=args.hparams_file, trainer=trainer)
         model = MegatronGPTModel.load_from_checkpoint(checkpoint_path, hparams_file=args.hparams_file, trainer=trainer)
+
+        # WAR to enable exporting FP8 scales from modules
+        # TODO: Need to find a cleaner way
+        for name, module in model.named_modules():
+            if hasattr(module, 'fp8'):
+                setattr(module, 'fp8', True)
+
```

Comment:
Let's leave this PR for docs only

Reply:
Oops that change was included accidentally. Reverting